.github/workflows/format_check.yml

```
@@ -34,3 +34,18 @@ jobs:
         run: flake8 tests --show-source
       - name: Lint moviepy
         run: flake8 moviepy --show-source
+  isort:
+    runs-on: ubuntu-latest
+    name: Isort
```

```suggestion
    name: isort
```
A bit of a nitpick, but unlike Black and Flake8, isort is always lowercase :)

---

Yep, looks so bad capitalized :+1: